packages/app/pages/Edit/organizationEdit.tsx

```diff
@@ -0,0 +1,282 @@
+/**
+ * This page displays the current information for an Organization in editable text fields. They can edit the
+ * information to their satisfaction and then submit their changes to update the Organization.
+ *
+ * @package: REA app
+ * @author:  Nicholas Roth <Lou3797>
+ * @version: 2018.4.30
+ * @since:   4/16/2018
+ */
+
+import * as React from "react";
+import "./organizationEdit.css";
+import getOrganizationById from "../../../ui-bindings/Organization/getOrganizationById";
+import { getValidation } from "../Common/common";
+
+/**
+ * Creates the EditForm for a given Organization.
+ */
+const OrganizationForm = getOrganizationById(({ organization, loading, error }) => {
```

Comment:
One possible way to do this (with a little more readability than arrays) would be to define your getValidation(loading, error) function as
```
export function getValidation(loading, error) {
if (error) {
   return {readyToRender: false, component: _error};
} else if (loading) {
   return {readyToRender: false, component: _loading};
} else {
  return {readyToRender: true};
}
}
```

Then your function here can be
```
let validationInfo = getValidation(loading, error);
if(!validationInfo.readyToRender) {
    return validationInfo.component;
} 
return <EditForm organization={organization}/>
```

It's not a huge difference, but it might make it a little smaller or easier to read. It's essentially doing it the exact same way as you are right now.

Reply:
I'll leave the amount of shrinking to each coder's discretion. For me, the if statement way would be quicker if I wanted to add any additional HTML.